probe/docker/container.go

```diff
@@ -277,12 +277,15 @@ func (c *container) ports(localAddrs []net.IP) report.StringSet {
 			continue
 		}
 		for _, b := range bindings {
-			if b.HostIP == "0.0.0.0" {
-				for _, ip := range localAddrs {
+			if b.HostIP != "0.0.0.0" {
```

Comment:
> Will it also be "0.0.0.0" if the host is using IPv6?

I've no idea. This PR is not really about making it work with IPv6, but making the reports smaller and easier to understand when you are using ipv4.  If you're using IPv6, you will get undefined behaviour.  This PR doesn't change that.